sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/orc/OrcScan.scala

```diff
@@ -40,4 +40,10 @@ case class OrcScan(
     OrcPartitionReaderFactory(sparkSession.sessionState.conf, broadcastedConf,
       dataSchema, fileIndex.partitionSchema, readSchema)
   }
+
+  override def supportDataType(dataType: DataType): Boolean = {
+    OrcDataSourceV2.supportDataType(dataType)
+  }
+
+  override def toString: String = "ORC"
```

Comment:
There is much more useful information in this class than just the file format name. This should use a `formatName` method instead so that `toString` can be used to show the object itself when debugging or logging.

Reply:
+1